README.Rmd

```diff
@@ -17,13 +17,33 @@ knitr::opts_chunk$set(
 
-This package estimates the time-varying reproduction number, growth rate, and doubling time using a range of open-source tools ([Abbott et al.](https://doi.org/10.12688/wellcomeopenres.16006.1)), and current best practices ([Gostic et al.](https://doi.org/10.1371/journal.pcbi.1008409)). It aims to help users avoid some of the limitations of naive implementations in a framework that is informed by community feedback and is actively supported. 
+## Summary
 
-It estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). Imputed infections are then mapped to observed data (for example cases by date of report) via a series of uncertain delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity. 
+EpiNow2 estimates the time-varying reproduction number, growth rate, and doubling time using a range of open-source tools ([Abbott et al.](https://doi.org/10.12688/wellcomeopenres.16006.1)), and current best practices ([Gostic et al.](https://doi.org/10.1371/journal.pcbi.1008409)). It aims to help users avoid some of the limitations of naive implementations in a framework that is informed by community feedback and is actively supported.
+
+Forecasting is also supported for the time-varying reproduction number, infections, and reported cases using the same generative process approach as used for estimation.
+
+<details> <summary> More details </summary>
+
+EpiNow2 estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). Imputed infections are then mapped to observed data (for example cases by date of report) via a series of uncertain delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity. 
 
 Uncertainty is propagated from all inputs into the final parameter estimates, helping to mitigate spurious findings. This is handled internally. The time-varying reproduction estimates and the uncertain generation time also give time-varying estimates of the rate of growth.
 
-The default model uses a non-stationary Gaussian process to estimate the time-varying reproduction number and then infer infections. Other options include:
+</details>
+
+<details> <summary> Models provided </summary>
+
+`{EpiNow2}` is designed to be used via a single function call to two functions:
+
+* [`epinow()`](https://epiforecasts.io/EpiNow2/reference/epinow.html): Estimate Rt and cases by date of infection and forecast these infections into the future.
```

Comment:
I was wondering about this too. Like which one should we be linking to here?

Reply:
I think there was an issue with CRAN doing this but I am not 💯

Reply:
We *could* update the relative to absolute links in the file prior to submission (in some kind of automated fashion, ideally) but it's not ideal.

> I was wondering about this too. Like which one should we be linking to here?

If we could use relative links then any README could link to the correct one, at least on the pkgdown site (though possibly not in the github web interface).

Reply:
Shall we just switch over and see how that goes for the next submission?

Reply:
so for relative links it would be something like
```suggestion
* [`epinow()`](reference/epinow.html): Estimate Rt and cases by date of infection and forecast these infections into the future.
```
right? We could then perhaps even bulk replace them with absolute links in the [README action](https://github.com/epiforecasts/EpiNow2/blob/main/.github/workflows/render-readme.yaml)?

Alternatively it might be possible to do this using `knitr::knit_expand()` and templates such as in https://github.com/epiverse-trace/packagetemplate/blob/main/.github/workflows/render_readme.yml
but I'm not sure how this plays with pkgdown.